server/server_test.go

```diff
@@ -20,16 +21,18 @@ import (
 )
 
 var (
+	peer                                                       *ipfslite.Peer
 	client                                                     pb.IpfsLiteClient
 	stringToAdd                                                string = "hola"
 	refFile, refLargeFile                                      *pb.Node
-	refSize                                                           int32
+	refSize                                                    int32
 	refNode0, refNode1, refNode2, refNode3                     *cbor.Node
 	refProtoNode0, refProtoNode1, refProtoNode2, refProtoNode3 *merkledag.ProtoNode
 )
 
 func TestSetup(t *testing.T) {
-	peer, err := newPeer()
+	var err error
+	peer, err = newPeer()
```

Comment:
For anyone looking over this, it might be worth looking into `newPeer()` to make sure it looks correct.